extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.js

```diff
@@ -164,4 +175,24 @@ export class AmpStoryInteractiveImgQuiz extends AmpStoryInteractive {
       setImportantStyles(el, {'--option-percentage': `${percentages[index]}%`});
     });
   }
+
+  /**
+   * Set the text color of the answer choice bubble to be readable and
+   * accessible according to the background color.
+   *
+   * @param {!Element} root
+   * @private
+   */
+  setBubbleColor_(root) {
```

Comment:
Nit: `setBubbleTextColor_` helps understand that you're just changing the text color and not the background color